From: Stephane Glondu <steph@glondu.net>
Date: Mon, 10 Feb 2020 07:49:09 +0100
Subject: Disable page_size test

---
 test/unix/test_lwt_bytes.ml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/unix/test_lwt_bytes.ml b/test/unix/test_lwt_bytes.ml
index 0bc62d8..3d5e397 100644
--- a/test/unix/test_lwt_bytes.ml
+++ b/test/unix/test_lwt_bytes.ml
@@ -773,10 +773,12 @@ let suite = suite "lwt_bytes" [
       Lwt.return check
     end;
 
+(*
     test "page_size" begin fun () ->
       let sizes = [4096; 16384; 65536] in
       Lwt.return (List.mem Lwt_bytes.page_size sizes)
     end;
+*)
 
     test "mincore buffer length = page_size * 2, n_states = 1"
       ~only_if:(fun () -> not Sys.win32) begin fun () ->
